packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -149,7 +156,9 @@ export async function apiFetchCustomer(): Promise<Customer> {
   );
 }
 
-export async function apiFetchPlanUpgradeEligibility(planId: string) {
+export async function apiFetchSubscriptionChangeEligibility(
```

Comment:
Could we change the name to `apiFetchPlanEligibility`, since it's use isn't strictly for subscription changes?